static/redfish/v1/JsonSchemas/OemManager/index.json

```diff
@@ -580,6 +590,26 @@
                 }
             },
             "type": "object"
+        },
+        "IBM": {
+            "additionalProperties": true,
+            "description": "Oem properties for IBM.",
+            "patternProperties": {
+                "^([a-zA-Z_][a-zA-Z0-9_]*)?@(odata|Redfish|Message)\\.[a-zA-Z_][a-zA-Z0-9_]*$": {
+                    "description": "This property shall specify a valid odata or Redfish property.",
+                    "type": [
+                        "array",
+                        "boolean",
+                        "integer",
+                        "number",
+                        "null",
+                        "object",
+                        "string"
+                    ]
+                }
+            },
+            "properties": {},
```

Comment:
Where is the USBCodeUpdateEnabled property? This still looks incomplete

Reply:
Done.
I saw that some properties were not added to the schema, so I didn't add USBCodeUpdateEnabled property. 
For example, the Certificates property : https://github.com/ibm-openbmc/bmcweb/blob/master/redfish-core/lib/managers.hpp#L1958  is not in the schema : https://github.com/ibm-openbmc/bmcweb/blob/master/static/redfish/v1/JsonSchemas/OemManager/index.json#L314

It doesn't matter. In the new patchset, I've added the USBCodeUpdateEnabled property to the schema.